src/Table/Table.jsx

```diff
@@ -98,19 +148,26 @@ Table.propTypes = {
   data: PropTypes.array.isRequired,
   filter: PropTypes.func,
   fixed: PropTypes.bool,
+  initialPage: PropTypes.number,
   initialSortState: tablePropTypes.sortState,
+  onPageChange: PropTypes.func,
   onSortChange: PropTypes.func,
+  pageSize: PropTypes.number,
+  paginated: PropTypes.bool,
   rowIDFn: PropTypes.func.isRequired,
 };
 
 Table.defaultProps = {
   filter: () => true,
+  onPageChange: () => {},
   onSortChange: () => {},
+  pageSize: DEFAULT_PAGE_SIZE,
 };
 
 Table.cssClass = {
   BODY: "Table--body",
   FIXED: "Table--fixed",
+  NO_DATA: "Table--no_data_cell",
```

Comment:
underscores are kinda uncommon for a CSS name, no big deal but `-` is more customary i think